examples/fortran/MarsCrustalThickness/meson.build

```diff
@@ -0,0 +1,12 @@
+TestMarsCrustalThickness = executable(
+  'MarsCrustalThickness',
+  'MarsCrustalThickness.f95',
+  link_with: shtools)
+
+test(
+  'MarsCrustalThickness',
+  TestMarsCrustalThickness,
+  args: [
+    meson.current_source_dir() / '../../ExampleDataFiles',
+    meson.current_source_dir() / 'input.txt',
+  ])
```

Comment:
At this point only for the fortran programs are being tested, but for the pyshtools distribution we should probably test the python components. Similar to what is done in the Makefile, I think that we should probably create a few test "suites" for: fortran, fortran-mp, fortran-no-timing, python, python-no-timing. These would be specified by adding the keyword `suite: ['fortran', 'fortran-no-timing']` to `test`.

I will probably just make separate PR on your branch in the next day or two (unless you get to this first!)